docker/build/README.md

```diff
@@ -6,7 +6,7 @@ Files in this directory demonstrate how to build and install the SGX SDK and PSW
 
 ###  Prerequisites
 1. Install [Docker and Compose](https://docs.docker.com/) and configure them properly following their respective installation guide.
-2. Install [SGX out-of-tree driver](https://github.com/intel/linux-sgx-driver). **Note**: See below to run with the DCAP driver or an SGX capable kernel.
+2. Install [SGX driver](https://github.com/intel/SGXDataCenterAttestationPrimitives/driver/linux). **Note**: See below to run with the legacy non-FLC SGX driver.
```

Comment:
Please use consistent terminology. We use "SGX driver" here but "DCAP driver" below.
We may need a subsequent PR to have this terms aligned with the Installation Guide, where there's a proposal to use the terms Flexible Launch Control Driver and Legacy Launch Control Driver.